core/templates/dev/head/components/forms/schema_editors/SchemaBasedHtmlEditorDirective.js

```diff
@@ -23,11 +23,29 @@ oppia.directive('schemaBasedHtmlEditor', [
         localValue: '=',
         isDisabled: '&',
         labelForFocusTarget: '&',
-        uiConfig: '&'
+        uiConfig: '&',
       },
       templateUrl: UrlInterpolationService.getDirectiveTemplateUrl(
         '/components/forms/schema_editors/' +
         'schema_based_html_editor_directive.html'),
-      restrict: 'E'
+      restrict: 'E',
+      link: function($scope, element, $attrs) {
+        var observer = new MutationObserver(function(mutations) {
```

Comment:
I would add some comments to the code explaining what is happening.

Reply:
Done.

Reply:
Doesn't seem done; please make sure to check GitHub before submitting your review reply.